SCIM_resource_type.json

```diff
@@ -1,9 +1,9 @@
 [
   {
     "schemas": ["urn:ietf:params:scim:schemas:core:2.0:ResourceType"],
-    "id": "Device",
-    "name": "Device",
-    "endpoint": "/Device",
+    "id": "Devices",
+    "name": "Devices",

```

Comment:
"id" and "name" should be kept singular "Device", ref https://datatracker.ietf.org/doc/html/rfc7643#section-8.6

For example, this is how "User" is shown in the above reference:
```json
"id": "User",
"name": "User",
"endpoint": "/Users",
```

Similar for Groups:
```json
"id": "Group",
"name": "Group",
"endpoint": "/Groups",
```

Reply:
This is correct. The resources are each singular but since an endpoint allows retrieval of more than one, the endpoint is plural.   Groups contains one or more Group resources.